relay-server/src/extractors/signed_json.rs

```diff
@@ -37,24 +37,26 @@ impl ResponseError for SignatureError {
     }
 }
 
+fn extract_header<'a>(
+    req: &'a HttpRequest<ServiceState>,
+    name: &'static str,
+) -> Result<&'a str, SignatureError> {
+    req.headers()
+        .get(name)
+        .ok_or(SignatureError::MissingHeader(name))
+        .and_then(|value| {
+            value
+                .to_str()
+                .map_err(|_| SignatureError::MalformedHeader(name))
+        })
+}
+
 impl<T: DeserializeOwned + 'static> FromRequest<ServiceState> for SignedJson<T> {
     type Config = ();
     type Result = ResponseFuture<Self, Error>;
 
     fn from_request(req: &HttpRequest<ServiceState>, _cfg: &Self::Config) -> Self::Result {
-        macro_rules! extract_header {
-            ($name:expr) => {
-                tryf!(req
-                    .headers()
-                    .get($name)
-                    .ok_or(SignatureError::MissingHeader($name))
-                    .and_then(|value| value
-                        .to_str()
-                        .map_err(|_| SignatureError::MalformedHeader($name))))
-            };
-        }
-
-        let relay_id: RelayId = tryf!(extract_header!("X-Sentry-Relay-Id")
+        let relay_id: RelayId = tryf!(tryf!(extract_header(req, "X-Sentry-Relay-Id"))
```

Comment:
This is an unrelated change without functional impact. Can we back this out into a separate PR, please?

Reply:
I had a suspicion you are going to say that :-) , placed it in a separate commit (I'll pick it out).